app/uk/gov/hmrc/exports/repositories/SubmissionRepository.scala

```diff
@@ -47,9 +47,19 @@ class SubmissionRepository @Inject() (val mongoComponent: MongoComponent)(implic
   override def classTag: ClassTag[Submission] = implicitly[ClassTag[Submission]]
   override val executionContext = ec
 
-  def addAction(mrn: String, newAction: Action): Future[Option[Submission]] = {
-    val filter = Json.obj("mrn" -> mrn)
-    val update = Json.obj("$addToSet" -> Json.obj("actions" -> Json.toJson(newAction)))
+  def addAction(uuid: String, action: Action): Future[Option[Submission]] = {
+    val filter = Json.obj("uuid" -> uuid)
+    val update = Json.obj("$addToSet" -> Json.obj("actions" -> Json.toJson(action)))
+    findOneAndUpdate(filter, update)
+  }
+
+  def addExternalAmendmentAction(uuid: String, action: Action): Future[Option[Submission]] = {
+    val filter = Json.obj("uuid" -> uuid)
+    val update = Json.obj(
+      "$addToSet" -> Json.obj("actions" -> Json.toJson(action)),
+      "$inc" -> Json.obj("latestVersionNo" -> 1),
```

Comment:
We shouldn't be setting the  `latestVersionNo` field to '1' here, it should be set to `latestVersionNo` + 1

Reply:
It's already doing that. The op here is `$inc` as you can see

Reply:
Ah ok, sorry I miss interpreted `$inc' to be 'include' rather than increment :stuck_out_tongue_winking_eye: All good then :+1: